tests/unit/classifier/LDA_unittest.cc

```diff
@@ -233,6 +203,32 @@ TEST(LDA, DISABLED_CheckProjection_SVD)
 	EXPECT_NEAR(+7.96084156, projection_SVD[9], epsilon);
 }
 
+// label type exception test
+TEST(LDA, num_classes_in_labels_exception)
+{
+	SGVector<float64_t> lab;
+	SGMatrix<float64_t> feat;
+
+	CDenseLabels* labels;
+	CDenseFeatures<float64_t>* features;
```

Comment:
For conciseness and better readiblity, write the variable declarations together with their initialization, for the ones  where it's possible.

Reply:
+1